cli/lib/util.js

```diff
@@ -16,6 +16,10 @@ const pkg = require(path.join(__dirname, '..', 'package.json'))
 const logger = require('./logger')
 const debug = require('debug')('cypress:cli')
 
+Promise.config({
+  cancellation: true,
```

Comment:
why is this necessary? I don't see us calling `promise.cancel()` anywhere?

Reply:
that should be taken out, left in there when I tried using bluebird timeouts